ee-feature-pack/galleon-shared/src/main/resources/modules/system/layers/base/jakarta/faces/api/main/module.xml

```diff
@@ -24,23 +24,50 @@
 
 <module xmlns="urn:jboss:module:1.9" name="jakarta.faces.api">
 
-    <resources>
-        <artifact name="${jakarta.faces:jakarta.faces-api}"/>
-    </resources>
-
     <dependencies>
         <module name="java.desktop"/>
         <module name="java.logging"/>
         <module name="java.sql"/>
         <module name="java.xml"/>
-        <module name="com.sun.jsf-impl"/>
         <module name="jakarta.enterprise.api" export="true"/>
         <module name="jakarta.servlet.api" export="true"/>
         <module name="jakarta.servlet.jsp.api" export="true"/>
         <module name="jakarta.servlet.jstl.api" export="true"/>
         <module name="jakarta.validation.api" export="true"/>
-        <module name="org.glassfish.jakarta.el" export="true"/>
         <module name="jakarta.websocket.api"/>
+
+        <module name="jakarta.faces">
+            <exports>
+                <include-set>
+                    <path name="jakarta/faces"/>
+                    <path name="jakarta/faces/annotation"/>
+                    <path name="jakarta/faces/application"/>
+                    <path name="jakarta/faces/component"/>
+                    <path name="jakarta/faces/component/behavior"/>
+                    <path name="jakarta/faces/component/html"/>
+                    <path name="jakarta/faces/component/search"/>
+                    <path name="jakarta/faces/component/visit"/>
+                    <path name="jakarta/faces/context"/>
+                    <path name="jakarta/faces/convert"/>
+                    <path name="jakarta/faces/el"/>
+                    <path name="jakarta/faces/event"/>
+                    <path name="jakarta/faces/flow"/>
+                    <path name="jakarta/faces/flow/builder"/>
+                    <path name="jakarta/faces/lifecycle"/>
+                    <path name="jakarta/faces/mock"/>
+                    <path name="jakarta/faces/model"/>
+                    <path name="jakarta/faces/push"/>
+                    <path name="jakarta/faces/render"/>
+                    <path name="jakarta/faces/validator"/>
+                    <path name="jakarta/faces/view"/>
+                    <path name="jakarta/faces/view/facelets"/>
+                    <path name="jakarta/faces/webapp"/>
```

Comment:
Have we done any checks to make sure that the number of classes before the change and after the change match?

Reply:
I have not verified the number of classes, but I pulled all of those package names directly from the source via `grep | cut | sort`, so it should be exhaustive.